.github/workflows/python_build.yml

```diff
@@ -0,0 +1,72 @@
+name: Build
```

Comment:
The tag release probably won't work in the context of an ASF repo anymore?

Reply:
Yeap, we will need to work out a packaging; the build of the wheels is imo still relevant, as it is not so easy in Rust (afai understood support for this is still a bit WIP). Building the manylinux was a feat.